pkg/auth/handler/webapp/authflowv2/viewmodels/preview.go

```diff
@@ -0,0 +1,6 @@
+package viewmodels
+
+const (
+	PreviewDummyPhoneNumber = "+85298765432"
```

Comment:
I am think if this phone number really exist.
Or we should use some text like `YOUR PHONE NUMBER`?
@fungc-io

Reply:
show `PHONE NUMBER` is ok